text/pdf_highlight_text.go

```diff
@@ -0,0 +1,135 @@
+/*
+ * Highlight text: Hightlight target texts inside the PDF file.
+ *
+ * Run as: go run pdf_highlight_text.go inputFile.pdf outputFile.pdf term
+ */
+
+package main
+
+import (
+	"fmt"
+	"os"
+	"strings"
+
+	"github.com/unidoc/unipdf/v3/common/license"
+	"github.com/unidoc/unipdf/v3/creator"
+	"github.com/unidoc/unipdf/v3/extractor"
+	"github.com/unidoc/unipdf/v3/model"
+)
+
+func init() {
+	err := license.SetMeteredKey(`put your license key here`)
```

Comment:
please copy from another example, we want to keep this the same everywhere, so new users can just use the same environment variable for trying things out